doc/ref/types.xml

```diff
@@ -385,7 +502,14 @@ attribute value, and just removing the value might leave the data
 structures in an inconsistent state. If necessary, a new object can be
 constructed.
 <P/>
-Several attributes have methods for more than one argument.  For example
+Each method that is installed for an attribute via <Ref Func="InstallMethod"/>
```

Comment:
In this paragraph I would quote the method names before their reference (4 instances).

Reply:
I don't understand what you mean with that.

Reply:
This was a question concerning the rendering of links which does not warrant further change (I misinterpreted the rendering).